lib/ex_unit/lib/ex_unit/case.ex

```diff
@@ -376,12 +377,14 @@ defmodule ExUnit.Case do
 
       @ex_unit_describe message
       @ex_unit_used_describes message
+      @ex_unit_describe_line __ENV__.line
```

Comment:
Instead of storing a new attribute, we can store the message and the line on ex_unit_describe. :)